ivy/functional/frontends/torch/tensor.py

```diff
@@ -1071,6 +1071,11 @@ def masked_fill_(self, mask, value):
         self.ivy_array = self.masked_fill(mask, value).ivy_array
         return self
 
+    def masked_select(self, mask):
+        x, mask = torch_frontend.broadcast_tensors(self, mask)
+        out = x.reshape(-1).index_select(0, mask.reshape(-1).nonzero().flatten())
+        return torch_frontend.tensor(out)
```

Comment:
```suggestion
        return torch_frontend.masked_select(self, mask)
```
Couldn't we just call into the existing torch frontend `masked_select` function with something like this?